katportalclient/client.py

```diff
@@ -1256,7 +1256,7 @@ def sensor_history(self, sensor_name, start_time_sec, end_time_sec,
             sample time in the result.
             Default: False.
         interval: int
-            The resampling interval in seconds. 0 to disable.
+            The resampling interval in seconds. 0 to disable. Currently NOT supported.
```

Comment:
@ajoubertza If plan to support interval we need to parse the result set correctly and add the extra fields that are present in the new katstore.

Reply:
I suggest this interval parameter is removed.  It wasn't in the previous version of this method.  Here are the old signatures, from the _master_ branch:
```
    def sensor_history(self, sensor_name, start_time_sec, end_time_sec,
                       include_value_ts=False, timeout_sec=300):
... 

    def sensors_histories(self, filters, start_time_sec, end_time_sec,
                          include_value_ts=False, timeout_sec=300):
```

We actually need a timeout parameter.  If it is possible to override the request timeout on the HTTP fetch, then please add it.  If not, then log a warning that the parameter is ignored.

For the `sensors_histories ` method that queries multiple sensors, the timeout is shared over all the requests.  See the implementation in master.